examples/audio/audio_music_stream.c

```diff
@@ -37,7 +41,10 @@ int main(void)
     {
         // Update
         //----------------------------------------------------------------------------------
-        UpdateMusicStream(music);   // Update music buffer with new stream data
+        if (!AUDIO_THREAD_MUSIC_UPDATE)
```

Comment:
This mechanism looks very weird from an user perspective, I'd prefer to avoid it and look for an alternative.